docs/projects/blank/overview.md

```diff
@@ -1,3 +1,21 @@
-# Blank Template project
+# Blank Starter Project
 
-// Information about the blank template project
+The Blank starter project is a great starting point for working on the SpatialOS GDK for Unity if you want a clean slate.
+
+Check out the [Blank starter project on Github](https://github.com/spatialos/gdk-for-unity-blank-project)!
+
+## What does it come with?
+
+It has the minimal feature set required to start developing:
+
+- A Unity project setup with the SpatialOS GDK for Unity.
+- Sample local and cloud deployment configurations.
+- An empty snapshot.
+- Client and server workers JSON configurations.
```

Comment:
"client-worker" and "server-worker" to use consistent terminology. (Changing it is very confusing for users who have a fragile grasp of our technology.)
Link to worker config docs.

Reply:
added full changes from above